test/archive_test_results_eks.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Should this live under `test/eks`? I'm not sure if we will end up with a `test/vagrant` at some point so `test/archive_test_results.sh` would move.

Reply:
I would rather have it here, as everything in `test/eks` is eks/k8s specific, while this is purely a jenkins job thing.